specs/extension_image.md

```diff
@@ -1,65 +1,79 @@
 ---
-title: Sysext - System Extension
+title: Extension Images
```

Comment:
The new name is better… But can you provide a redirect from the name to the new? I'm sure that there are links out there to the old ULR.

Reply:
done